aten/src/ATen/native/Convolution.cpp

```diff
@@ -1286,14 +1286,14 @@ std::tuple<Tensor,Tensor,Tensor> _convolution_double_backward( const c10::option
       if (weight.is_cuda()) {
         weight = weight.contiguous();
       }
-      ggO = at::_convolution(ggI, weight, Tensor(), params.stride, params.padding, params.dilation, params.transposed, params.output_padding, params.groups, params.benchmark, params.deterministic, params.cudnn_enabled, params.allow_tf32);
```

Comment:
so currently _convolution_double_backward takes its own set of 4 bool flags (benchmark, deterministic, cudnn, tf32), and you're changing to call at::convolution() which does not take these flags instead inferring their values from 
ctx.benchmarkCuDNN(), deterministic, ctx.userEnabledCuDNN(), ctx.allowTF32CuDNN()

Is this actually equivalent, e.g. is every call site of convolution_double_backwards() already using 
ctx.benchmarkCuDNN(), deterministic, ctx.userEnabledCuDNN(), ctx.allowTF32CuDNN()

or is this introducing a possible behavior change?

Reply:
seems like in derivatives.yaml there are 2 flavors-
regular convolution_backward maps to this, which is close but not exactly equivalent to what at::convolution will do:
at::globalContext().benchmarkCuDNN(), 
at::globalContext().deterministicCuDNN() || at::globalContext().deterministicAlgorithms(), at::globalContext().userEnabledCuDNN(), 
at::globalContext().allowTF32CuDNN(),

but convolution_backward_overrideable and _slow_conv2d_backward.output_mask pass 
false, false, false, false
for these flags.

Reply:
Some thoughts:
* You are correct that the long term goal is to remove `convolution_backward_overrideable` entirely and have XLA implement `convolution` / `convolution_backward` instead. The values of the cuDNN flags don't matter to XLA I believe.
* Note that `convolution_backward` and `convolution` define these flags the same way since
```deterministic = at::globalContext().deterministicCuDNN() || at::globalContext().deterministicAlgorithms()```
* Outside of loading & running a previously serialized TorchScript model, it's not possible in the public API to run forward and backward with different flag settings without changing them in the global context. This is relevant because it makes it unlikely that these flag values will be different across layers within the same model.
* In talking with @ngimel and @albanD, I think it's okay not to require the same flag values across forward / backward pairs (including `convolution_backward` / `_convolution_double_backward`) but rather just query the global context for the backward pass. This allows for running the forward under cuDNN and the backward with cuDNN disabled, for example, by modifying the global context between forward and backward passes.

TL;DR: @desertfire I think we can move forward with this PR, but I'd like to see the 4 cuDNN flags removed from the `_convolution_double_backward` signature to avoid being misleading about where the values are coming from.